pandas/core/indexes/category.py

```diff
@@ -42,26 +42,44 @@
     typ='method', overwrite=True)
 class CategoricalIndex(Index, accessor.PandasDelegate):
     """
-    Immutable Index implementing an ordered, sliceable set. CategoricalIndex
-    represents a sparsely populated Index with an underlying Categorical.
+    Immutable index implementing an ordered, sliceable set. CategoricalIndex
+    represents a sparsely populated index with an underlying
+    :class:`Categorical`.
+
+    `CategoricalIndex`, like `Categorical` can only take on a limited,
+    and usually fixed, number of possible values (`categories`). Also,
+    like `Categorical`, it might have an order, but numerical operations
+    (additions, divisions, ...) are not possible.
 
     Parameters
     ----------
-    data : array-like or Categorical, (1-dimensional)
-    categories : optional, array-like
-        categories for the CategoricalIndex
-    ordered : boolean,
-        designating if the categories are ordered
-    copy : bool
+    data : list-like
+        The values of the categorical. If categories are given, values not in
+        categories will be replaced with NaN.
+    categories : index-like, optional
+        The categories for the categorical. Items need to be unique.
+        If the categories are not given here, then they must be provided
+        in `dtype`.
+    ordered : bool, optional
+        Whether or not this categorical is treated as an ordered
+        categorical. If not given here or in `dtype`, the resulting
+        categorical will be unordered.
+    dtype : CategoricalDtype or the string "category", optional
+        If :class:`CategoricalDtype`, cannot be used together with
+        `categories` or `ordered`.
+
+        .. versionadded:: 0.21.0
```

Comment:
Does this render? Might need a blank line thereafter

Reply:
Yes, it renders. See e.g. https://pandas.pydata.org/pandas-docs/stable/generated/pandas.Series.drop.html, where under ``dtype`` there also is a no blank line.

Reply:
Indeed, only a blank line before it is needed (because in this case the next line is a different block)